model/api/src/main/java/org/jboss/aerogear/unifiedpush/dao/VariantMetricInformationDao.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright Red Hat, Inc., and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.aerogear.unifiedpush.dao;
+import org.jboss.aerogear.unifiedpush.api.VariantMetricInformation;
+
+
```

Comment:
ok -

On Tue, Apr 14, 2015 at 11:00 AM, Sebastien Blanc <notifications@github.com>
wrote:

> In
> model/api/src/main/java/org/jboss/aerogear/unifiedpush/dao/VariantMetricInformationDao.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523#discussion_r28310578>
> :
>
> > + * Licensed under the Apache License, Version 2.0 (the "License");
> > + * you may not use this file except in compliance with the License.
> > + * You may obtain a copy of the License at
> > + *
> > + *     http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +package org.jboss.aerogear.unifiedpush.dao;
> > +import org.jboss.aerogear.unifiedpush.api.VariantMetricInformation;
> > +
> > +
>
> @matzew <https://github.com/matzew> +1 I think it would make sense to
> rebase this PR on the JMS PR
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523/files#r28310578>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf